### ISSUE TITLE
name unnamed elements for better TalkBack screen reader in Delta Chat support

### ISSUE THREAD

**Issue by OzancanKaratas** [deltachat/deltachat-android#1029]
If an issue wer opened for this topic, I apologize. I want to support TalkBack in Delta Chat. Can you help me? Please open a branch. Let's do together.

**Comment by r10s** [comment]
for everyone who wants to try this out:

- talkback can be enabled in the android settings
- once enabled, a single tap on an item reads that item, a double tap performs the item action
- for resume/continue, press volume up+down at the same time

i just played around, so, in general, things work, at least comparable with eg. the android system.

but there are some missing button names, i noticed that esp. for the "back" button, but probably, there are some more.

**Comment by OzancanKaratas** [comment]
The problems you mentioned here available also in iPhone.

**Comment by cyBerta** [comment]
@OzancanKaratas are you interested in reporting the main problems for android wrt screenreaders here in this ticket? I might help us to fix the worst screen reading UX issues faster.

**Comment by OzancanKaratas** [comment]
Some elements are not labeled for screen readers. I don't have an Android device, I cannot test it. I think @r10s should interest for this.

**Comment by r10s** [comment]
as in general, this seems to work i am closing this. i was also reported by some blind people via mail that this in general works.

we can open concrete issues as needed, eg. if some element is not names.
